src/storage/gpu_swap_prefetch.cc

```diff
@@ -7,15 +7,15 @@
 #include <dmlc/logging.h>
 #include <mxnet/gpu_swap_history.h>
 #include <mxnet/swap.h>
-#include <mxnet/gpu_swap_prefetch.h>
+#include "./gpu_swap_prefetch.h"
 
 
 namespace mxnet {
 
 Prefetch::Prefetch() {
   start_prefetching_ = false;
   stop_prefetching_ = false;
-  algorithm_ = dmlc::GetEnv("PREFETCH_ALGORITHM", 0);
+  prefetch_algorithm_ = dmlc::GetEnv("PREFETCH_ALGORITHM", 0);
```

Comment:
Like swap_algorithm_, we should let this environment variable to be a string environment variable. If we are going to have several algorithms, it will be more clear.